File: python/format.py
#!/Library/Frameworks/Python.framework/Versions/3.8/bin/python3

import re
import sys

def main():
    file_path = sys.argv[1]
    with open(file_path) as file:
        text = file.read()

    text = re.sub(r'^\s+', "", text)
    text = re.sub(r'\s+', " ", text)
    text = re.sub(r'\s+$', "", text)
    
    i, j = 0, 69

    while j < len(text):
        print(text[i:j])
        i += 69
        j += 69
        
    print(text[i:len(text)])

if __name__ == "__main__":
    main()

File: ruby/format.rb
#!/Users/benjamin/.rvm/rubies/ruby-2.6.3/bin/ruby

def main
  file_path = ARGV[0]
  text = File.open(file_path) { |file| file.read }

  text.gsub! %r{^\s+}, ""
  text.gsub! %r{\s+}, " "
  text.gsub! %r{\s+$}, ""

  i, j = 0, 68

  while j < text.length
    puts text.slice i..j
    i += 69
    j += 69
  end

  puts text.slice i..text.length - 1
end

if __FILE__ == $0
  main
end
